src/Organic/Organic.php

```diff
@@ -277,6 +284,31 @@ public function init( $apiUrl = null, $cdnUrl = null ) {
         add_action( 'save_post', [ $this, 'handleSavePostHook' ], 10, 3 );
     }
 
+    /**
+     * @param OrganicSdk $sdk
+     */
+    public function initSentry( \Organic\SDK\OrganicSdk $sdk ) {
+        if ( $this->environment != 'PRODUCTION' ) {
+            return;
+        }
+        try {
+            $config = $sdk->queryWordPressConfig();
+        } catch ( \Exception $e ) {
+            self::captureException( $e );
+        }
+        if ( ! empty( $config ) ) {
+            $sentryDSN = $config['sentryDsn'];
+            if ( ! empty( $sentryDSN ) ) {
+                \Sentry\init(
```

Comment:
> This will overwrite the global setup of Sentry for the current site

Ah, duh, I see 🤦 . This clarifies why references to Sentry were in the codebase without it being a requirement. I (obviously) didn't think of the interaction with any existing Sentry on the client WordPress. Thank you!